tests/integration/transactions/test_account_delete.py

```diff
@@ -28,7 +29,6 @@ async def test_all_fields(self, client):
             account_delete, WALLET, check_fee=False
         )
         self.assertEqual(response.status, ResponseStatus.SUCCESS)
-        WALLET.sequence += 1
```

Comment:
Why was this line deleted?

Reply:
The account delete doesn't actually work, because it requires 256 ledgers to go by. So the sequence number doesn't get incremented.